dags/process_md_1c_to_pg/libs/transform.py

```diff
@@ -16,6 +16,45 @@ def transform_data(in_fp: str, out_dp: str, src_map: dict, dest_map: dict, file_
         if col not in df.columns:
             df[col] = None
     df = df[dest_columns]
+
+    if 'ic' in df.columns:
+        df = df[df['ic'].fillna('').astype(str).str.strip().ne('')]
+    if 'items_type' in df.columns:
+        df = df[~df['items_type'].astype(str).str.strip().str.lower().isin(['total', 'итого'])]
+
+    if 'deletion_mark' in df.columns:
+        df['deletion_mark'] = (
+            df['deletion_mark']
+            .astype(str)
+            .str.strip()
+            .str.lower()
+            .map({'true': True, 'false': False, '1': True, '0': False, 'да': True, 'нет': False, 'y': True, 'n': False})
```

Comment:
Судя по содержимому файла `products_1c.xlsx`, поле `deletion_mark` может еще принимать значения "Yes"/"No"

Reply:
Тут, кстати, код можно чуть короче сделать:
```suggestion
            .isin(['true', '1', 'да', 'y', 'yes'])
```